openedx_webhooks/templates/github_community_pr_comment.md.j2

```diff
@@ -1,18 +1,32 @@
 Thanks for the pull request, @{{ user }}!
 
-### What's next?
+{% if owner %}
+This repository is currently maintained by @{{ owner }}.
+{% else %}
+{% if lifecycle == 'production' %}
+This repository has no maintainer (yet).
+{% else %}
+This repository is currently unmaintained.
+{% endif %}
+<details><summary>#### :radio_button: Find a technical reviewer</summary>
```

Comment:
Once you start using the HTML detail element, the markdown `#` is no longer respected. You should used `<h4>` instead.  See an example of how this plays out in https://github.com/openedx/webhook-test-repo/pull/18

Reply:
I've applied your suggestion. Thanks!